next.config.js

```diff
@@ -4,6 +4,11 @@ module.exports = {
   },
   async redirects() {
     return [
+      {
+        source: '/cfp',
+        destination: '/',
+        permanent: true,
```

Comment:
Shouldn't be `permanent: false`? It's a temporary redirect until next year :P

Reply:
next year :P

```suggestion
        permanent: false,
```